components/ILIAS/IndividualAssessment/PRIVACY.md

```diff
@@ -0,0 +1,116 @@
+# Individual Assessment Privacy
+
+Disclaimer: This documentation does not warrant completeness or correctness. Please
+report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+### General Information
+
+The Individual Assessment is intended as a performance and skill evaluation tool.
+The submitted data can shape a person's career. Therefore, some data is intentionally
+impossible to delete or change for most or all user roles. This makes it more likely
+that Individual Assessment records are acceptable as proof in court.
+
+### Integrated Services
+
+- The Individual Assessment component employs the following services, please consult
+the respective privacy.mds:
+    - [Metadata](../MetaData/Privacy.md)
+    - [AccessControl](../AccessControl/PRIVACY.md)
+    - [Object](../ILIASObject/PRIVACY.md)
+    - [InfoScreen](../InfoScreen/PRIVACY.md)
+- Users from [Course](../Course/PRIVACY.md) and [Group](../Group/PRIVACY.md) can be
+  added to an Individual Assessment. Even after a person's membership to the original
+  object has been removed, it may be obvious where the Users came from.
+- An Individual Assessment may be part of another object's Learning Progress
+  [Tracking](../Tracking/PRIVACY.md).
+
+## Data being stored
+
+- **Participants**: Adding users as Participants to the Individual Assessment references
+  their User object by ID.
+- **Examiner**: Examiners are users with the permission to grade a Participant. Their
```

Comment:
I'm not sure I agree with this labeling. There is a lack of consistency imo.
For me, Location and Time, Record Note, Grading etc. are "stored data", but Participants and Examiners are not. I can see what you mean, but maybe it would be better to turn it around and write something like "User ID": The user ID of Examinator is stored with the individual record. Examinators are users with the authorization to grade a participant." That way, everyone can immediately see **what data** is stored and then read the explanation.

Reply:
I agree. But I suggest to divide user ID for Participants and for Examiners:
**User ID of Participants**: Adding users as Participants to the Individual Assessment references their User object by ID.
**User ID of examinier**: Examiners are Users with the permission to grade a Participant. Their User ID is stored with the individual record.

Reply:
User ID of Participants: Adding accounts as Participants to the Individual Assessment references their User object by ID.
User ID of Examiners: Examiners are accounts with the permission to grade a Participant. Their User ID is stored with the individual record.

Reply:
I have changed to "Examiners are users that have graded one or more participants. Their User ID is stored with the individual record of some participant.". The actual permission at any given moment isn't exactly what counts. My ID won't be stored just because I have the permission. It will be stored if I actually grade a user. Also, it will still be stored if I lose that permission later on.